Lib/test/test_dbm_gnu.py

```diff
@@ -144,6 +144,13 @@ def test_nonascii_filename(self):
             self.assertTrue(b'key' in db)
             self.assertEqual(db[b'key'], b'value')
 
+    def test_nonexising_file(self):
+        unlink(filename)
```

Comment:
Which filename should be there which is guaranteed to be non-existing? I think we should use a file which is under our control and thereby I re-used the filename variable.
The unlink is a very last resort to ensure that the file won't exist, this would normall not remove any file as it is already removed in tearDown.

Reply:
Using something like `gdbm.open('nonexisting-file')` is enough.